src/Adapter/Product/CommandHandler/RemoveAllCustomizationFieldsFromProductHandler.php

```diff
@@ -51,23 +52,23 @@ final class RemoveAllCustomizationFieldsFromProductHandler implements RemoveAllC
     private $productProvider;
 
     /**
-     * @var ProductUpdater
+     * @var ProductPersister
```

Comment:
```suggestion
     * @var ProductCustomizationFieldUpdater
```

And you can remove the useless `use` statement for `ProductPersister`